lib/Rex/Group/Lookup/XML.pm

```diff
@@ -33,7 +33,7 @@ use Rex -base;
 require Exporter;
 use base qw(Exporter);
 use vars qw(@EXPORT);
-XML::LibXML->require;
+use XML::LibXML;
```

Comment:
What we can do is, that we provide a package where rex is bundled with perl, so that you can just run it on every glibc compatible linux system.